### DISCUSSION TITLE
Save Image As Not Working As Expected

### DISCUSSION THREAD

**Discussion by tengkuizdihar** [tauri-apps/tauri#7514] [Q&A]
If for example my application are using tiptap editor and it has a "save image as" option when I right click an image, why does it not popup a save file dialogue when I did it?

**Comment by FabianLars** [comment]
Could you open an issue about this instead? Please include the platforms you tested this on and their respective behavior. If you could provide a minimal reproduction example that would be nice too (but not a requirement for this imo)

**Reply by tengkuizdihar** [reply]
Will do (Soon™), I'm on NixOS right now.
